### PR TITLE
Update Shuffle

### PR DESCRIPTION
**Pull request by passerbyloo**
according to https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle

`checks if j ≠ i may be omitted`, and also remove checking logic will optimise performance(tested by 100000 times random).

### Checklist

- [x] I've looked at the [contribution guidelines](https://github.com/raywenderlich/swift-algorithm-club/blob/master/.github/CONTRIBUTING.md).
- [x] This pull request is complete and ready for review.

### CODE REVIEW

File: Shuffle/README.markdown [hunk 1]
```diff
@@ -96,9 +96,6 @@ public func shuffledArray(_ n: Int) -> [Int] {
   var a = [Int](repeating: 0, count: n)
   for i in 0..<n {
     let j = Int.random(in: 0...i)
-    if i != j {
-      a[i] = a[j]
```
Comment (richard-ash) [inline Shuffle/README.markdown]: Removing this line breaks the algorithm. You're right the check `if i != j` is unnecessary however you still need the line `a[i] = a[j]`

File: Shuffle/Shuffle.swift [hunk 2]
```diff
@@ -30,9 +30,6 @@ public func shuffledArray(_ n: Int) -> [Int] {
   var a = Array(repeating: 0, count: n)
   for i in 0..<n {
     let j = random(i + 1)
-    if i != j {
-      a[i] = a[j]
```
Comment (richard-ash) [inline Shuffle/Shuffle.swift]: Same here

File: Shuffle/README.markdown (hunk 1 above)
Comment (passerbyloo) [reply Shuffle/README.markdown]: updated

**Comment by richard-ash** [comment]
Cool, thanks for the update 🙂

**Pull request merged**